uwblueprint/community-fridge-kw
# feat: Add volunteer to sign up flow

## Brief description. What is this change? 

### [Add volunteer role in sign up flow](https://www.notion.so/uwblueprintexecs/Add-volunteer-role-in-sign-up-flow-559055affb464b619ab81f7127a89951)



## Implementation description. How did you make this change?
* Added a new step to the signup flow for users to choose `Donor/Volunteer` account type
* Remove `business name` for volunteers
* Make `phone number` not mandatory for volunteers
* Added a mandatory terms and conditions step for volunteers
* Made UI changes to match Figma designs



## Steps to test
1. Go through sign up flow as volunteer - ensure correct fields are shown
2. Go through sign up flow as donor - should be same as before
3. Check mobile and desktop view

https://user-images.githubusercontent.com/69697180/155259057-1a54f035-931c-4191-a0bc-0725cd552e8d.mov
https://user-images.githubusercontent.com/69697180/155259192-e3290214-8393-4a7e-a71b-7cdf936bda1f.mov



## Checklist
